Sources/Public/Animation/LottieView.swift

```diff
@@ -189,6 +189,14 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     return copy
   }
 
+  /// Returns a copy of this view with the given `LottieCompletionBlock` that is called
+  /// when an animation finishes playing.
+  public func animationDidFinish(_ animationCompletionHandler: LottieCompletionBlock?) -> Self {
+    var copy = self
+    copy.animationCompletionHandler = animationCompletionHandler
+    return copy
+  }
```

Comment:
Should we chain them?
```suggestion
  public func animationDidFinish(_ animationCompletionHandler: LottieCompletionBlock?) -> Self {
    var copy = self
    copy.animationCompletionHandler = {
      animationCompletionHandler?()
      copy.animationCompletionHandler?()
    }
    return copy
  }
```

Reply:
sure, seems reasonable